### ISSUE TITLE
not make mods implement the api so users can use the modifier keys.

### ISSUE THREAD

**Issue by dmodoomsirius** [mod-warriors/NotEnoughKeys#46]
I think we should make it so we can allow users without mods implementing the api as that would be a pain for any mod and if there are new ones with key binds they would have to implement the api so their users can use the modifier keys with their key binds.

**Comment by TheTemportalist** [comment]
This was always the goal, the API was just a temporary fix until a better way can be figured out.

**Comment by DanHolli** [comment]
please add this as I currently have 116 mods on 1.8.9 and still adding and none of the mods show up seperatly

**Comment by Kagenokami** [comment]
Yeah, so what I am seeing is that, as it stands, it is impossible to really USE this since you cannot slip the api into a mod forcefully without great effort...Any word on the progress on making it work without the use of the API?
